app/src/main/res/xml/launcher_dock_preferences.xml

```diff
@@ -1,89 +1,105 @@
+                android:defaultValue="true"
+                android:persistent="true" />
 
-        <SwitchPreference
-            android:key="pref_isHotseatTransparent"
-            android:title="@string/transparent_hotseat_pref_title"
-            android:defaultValue="false"
-            android:persistent="true" />
+            <SwitchPreference
+                android:key="pref_hotseatShowPageIndicator"
+                android:title="@string/hotseat_show_page_indicator"
+                android:defaultValue="true"
+                android:persistent="true" />  
+        </android.support.v7.preference.PreferenceCategory>
+    
+        <android.support.v7.preference.PreferenceCategory
+            android:key="prefCat_dockIcons"
+            android:title="@string/pref_category_icons">
+            
+            <ListPreference
+                android:key="pref_numHotseatIcons"
+                android:title="@string/num_hotseat_icons_pref_title"
+                android:entryValues="@array/numHotseatIconsValues"
+                android:entries="@array/numHotseatIconsTexts"
+                android:defaultValue="@string/generalDefault"
+                android:summary="%s"
+                android:persistent="true" />
 
-        <SwitchPreference
-            android:key="pref_hideHotseat"
-            android:title="@string/hide_hotseat_pref_title"
-            android:defaultValue="false"
-            android:dependency="pref_isHotseatTransparent"
-            android:persistent="true" />
+            <ch.deletescape.lawnchair.preferences.SeekbarPreference
+                android:key="pref_hotseatIconScale"
+                android:title="@string/icon_scale_pref_title"
+                app:minValue="0.3"
+                app:maxValue="1.5"
+                app:summaryFormat="%.0f%%"
+                app:summaryMultiplier="100"
+                app:defaultSeekbarValue="1.0"
+                app:steps="120"
+                android:persistent="true" />     
+        </android.support.v7.preference.PreferenceCategory>
+    
+        <android.support.v7.preference.PreferenceCategory
+            android:key="prefCat_dockMisc"
+            android:title="@string/pref_category_misc">
+            
+            <SwitchPreference
+                android:key="pref_hotseatShouldUseExtractedColors"
+                android:title="@string/hotseat_uses_extracted_colors_pref_title"
+                android:summaryOn="@string/hotseat_uses_extracted_colors_pref_on"
+                android:summaryOff="@string/hotseat_uses_extracted_colors_pref_off"
+                android:defaultValue="true"
+                android:persistent="true" />
 
-        <SwitchPreference
-            android:key="pref_hotseatShowArrow"
-            android:title="@string/hotseat_show_arrow"
-            android:defaultValue="true"
-            android:persistent="true" />
+            <SwitchPreference
+                android:key="pref_isHotseatTransparent"
```

Comment:
Doesn't it require to also change the preference key in the code?

Reply:
He didn't change preference keys, he just moved some preferences around, this one used to be located a bit further up in the file.

Reply:
Yeah, it doesn't need to be changed. It's already added up a bit like what Till said,

Reply:
Oh I see, stupid git diff.